src/app/machines/views/MachineList/MachineListTable/types.ts

```diff
@@ -0,0 +1,63 @@
+import type { ReactNode } from "react";
+
+import type { ValueOf } from "@canonical/react-components";
+import type { MainTableCell } from "@canonical/react-components/dist/components/MainTable/MainTable";
+
+import type { SortDirection } from "app/base/types";
+import type { MachineColumns } from "app/machines/constants";
+import type { GetMachineMenuToggleHandler } from "app/machines/types";
+import type {
+  Machine,
+  MachineStateListGroup,
+  FetchGroupKey,
+} from "app/store/machine/types";
+
+export type Props = {
```

Comment:
I think naming the type `Props` was okay when it was a local type, it looks too generic now that it's being shared between files and can be mistaken for another prop type. We could call it `MachineListTableProps`

Reply:
Agreed, marking this as `Code +1 (with changes)`

Reply:
Good point, will update!